test/source/_posts/2016-08-16-indented-link-references.markdown

```diff
@@ -0,0 +1,17 @@
+---
+---
+
+This is the first paragraph. It [have][link_0] [lots][link_1] [of][link_2]
+[links][link_3].
+
+This is the second paragraph. It have a sample code that should not be
```

Comment:
It's just nitpicking but since it's not merged yet I didn't see the harm --- and I figured easy to read tests acts as their own sort of documentation.